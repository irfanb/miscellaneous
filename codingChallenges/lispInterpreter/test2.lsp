1
"Hello, Coding Challenges"
:CC
(format t "Hello, Coding Challenge World World")
(defun hello () "Hello, Coding Challenge World")
