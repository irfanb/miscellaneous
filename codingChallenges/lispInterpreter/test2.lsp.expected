found number 1
found STRINGATOM "Hello, Coding Challenges"
found SYMBOLATOM :CC
found BEGINEXPRESSION (
found id format
found id t
found STRINGATOM "Hello, Coding Challenge World World"
found ENDEXPRESSION )
found BEGINEXPRESSION (
found id defun
found id hello
found BEGINEXPRESSION (
found ENDEXPRESSION )
found STRINGATOM "Hello, Coding Challenge World"
found ENDEXPRESSION )
